src/applications/vaos/appointment-list/components/ConfirmedAppointmentDetailsPage/VideoLink.jsx

```diff
@@ -15,7 +15,8 @@ export default function VideoLink({ appointment }) {
   } else if (appointment.videoData.kind === VIDEO_TYPES.clinic) {
     return (
       <span>
-        You must join this video meeting from the VA location listed below.
+        You must join this video meeting from the VA location{' '}
+        {hasFacility ? 'listed below' : 'where the appointment was scheduled'}.
```

Comment:
I will add a test for this when https://github.com/department-of-veterans-affairs/vets-website/pull/17313 is merged, since it changes the logic for what id is used.